deeprank_gnn/models/contact.py

```diff
@@ -53,25 +54,29 @@ def vanderwaals_potential(self) -> float:
 
     @property
     def residue1(self) -> Residue:
-        return self.item1
+        return self.residue1
 
     @property
     def residue2(self) -> Residue:
-        return self.item2
+        return self.residue2
```

Comment:
I think this is the right attribute (item2 does not exist)

Reply:
I suppose _residue2 will work, but not residue2, since that would be a circular call.